src/genetic_testing/assay_target/primer_design.py

```diff
@@ -82,7 +82,8 @@ def iter_window(ref, window, slide_ind, aln, aln_l):
     j = 0
     seq_l = []
     while j < len(ref) - window:
-        primer_seq = ref[j : j + window]
+        startIdx = j + 1
```

Comment:
It may be helpful to add a comment around this logic. As someone unfamiliar with the code, I don't have a good understanding about what is going on here.